### PR TITLE
fix pledge in test cluster

### PR DESCRIPTION
**Pull request by disassembler**

### CODE REVIEW

File: nix/supervisord-cluster/default.nix [hunk 1]
```diff
@@ -336,7 +336,7 @@ let
     ${pkgs.python3Packages.supervisor}/bin/supervisord --config ${supervisorConfig} $@
     while [ ! -S $CARDANO_NODE_SOCKET_PATH ]; do echo "Waiting 5 seconds for bft node to start"; sleep 5; done
     echo "Transfering genesis funds to pool owners, register pools and delegations"
-    cardano-cli shelley transaction submit \
+    cardano-cli shelley transaction submit --shelley-mode \
```
Comment (newhoggy) [inline nix/supervisord-cluster/default.nix:339]: Does our test cluster run in Shelley only mode?

Comment (dorin100) [reply nix/supervisord-cluster/default.nix:339]: @disassembler can you please help us to merge this PR? The scripts need this in order to provide accurate results. Thank you.

**Pull request merged**